HW3.postman_collection.json

```diff
@@ -0,0 +1,133 @@
+{
+	"info": {
+		"_postman_id": "0860eec4-d24b-49d5-9c82-45a6ebb7146f",
+		"name": "HW_3",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "21168131"
+	},
+	"item": [
+		{
+			"name": "Pokemon_1",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"const jsonObj = pm.response.json();\r",
+							"const nameArray = [];\r",
```

Comment:
forEach() loop plus nameArray.push() construction may be replaced using [Array.prototype.find()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find).